src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp

```diff
@@ -382,7 +382,13 @@ int os::extra_bang_size_in_bytes() {
 
 extern "C" {
   int SpinPause() {
-    return 0;
+    using spin_wait_func_ptr_t = void (*)();
+    spin_wait_func_ptr_t func = CAST_TO_FN_PTR(spin_wait_func_ptr_t, StubRoutines::aarch64::spin_wait());
+    if (func == nullptr) {
+      return 0;
+    }
```

Comment:
It's better simply to give `_spin_wait` a default value that points to a `ret` instruction.

Reply:
Done